device-types/MikroTik/CSS610-8G-2S+IN.yaml

```diff
@@ -0,0 +1,47 @@
+---
+manufacturer: MikroTik
+model: CSS610-8G-2S+IN
+part_number: CSS610-8G-2S+IN
+slug: mikrotik-css610-8g-2s-in
+is_full_depth: false
+airflow: passive
+u_height: 1
```

Comment:
Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Reply:
Hi @danner26, thanks for checking again. I previously stated that I was unable to acquire the weight information from MikroTik specifications and any documentation like datasheets. Unfortunately this still holds true.